docs/pages/workflow/legacy-expo-cli.md

```diff
@@ -0,0 +1,1079 @@
+---
+title: Legacy Global Expo CLI
+maxHeadingDepth: 4
```

Comment:
Is there a better place for deprecated docs?

Reply:
Hmm, maybe https://docs.expo.dev/archive/ or "Deprecated" group at the end of guides (https://docs.expo.dev/guides/)?

Reply:
You can't navigate to archive, and you still need the legacy CLI for some commands, so Deprecated section in guides seems better.

Reply:
`pages/archive`